lnd_test.go

```diff
@@ -99,12 +101,109 @@ func testBasicChannelFunding(net *networkHarness, t *testing.T) {
 	assertTxInBlock(block, closingTxid, t)
 }
 
+func testChannelBalance(net *networkHarness, t *testing.T) {
+	ctxb := context.Background()
+
+	openChannel := func(alice *lightningNode, bob *lightningNode, amount btcutil.Amount) *lnrpc.ChannelPoint {
```

Comment:
I see what you're aiming for here (code re-use from the `testBasicChannelFundingTest`), but as implemented we don't really gain much since this function is only used once within this test. 

How about moving `openChannel` and `closeChannel` to a package-level function rather than a function closure. With this change, the functions can be used across other test cases within `lnd_test.go` (including `testBasicChannelFundingTest`).